kartothek/io_components/docs.py

```diff
@@ -119,8 +119,8 @@
         kinds of predicates that are possible using boolean logic.
 """,
     "secondary_indices": """
-    secondary_indices: List[str]
-        A list of columns for which a secondary index should be calculated.
+    secondary_indices: Set[str]
```

Comment:
Please don't change this as
1. Technically speaking this would be a breaking change
2. We do not accept/enforce sets as input anywhere else
3. We neither validate nor cast anything to sets atm

Reply:
I dropped the commit